lib/src/document.dart

```diff
@@ -13,6 +13,7 @@ class Document {
   final ExtensionSet extensionSet;
   final Resolver linkResolver;
   final Resolver imageLinkResolver;
+  final bool guardHtml;
```

Comment:
Hmm, `guardHtml` doesn't intuitively tell me what it does, or which way is normal. How about `encodeHtml` which would default to true?